cmd/node/main.go

```diff
@@ -367,6 +375,19 @@ func startNode(ctx *cli.Context, log *logger.Logger, version string) error {
 	}
 	log.Info("Starting with public key: " + factory.GetPkEncoded(pubKey))
 
+	destinationShardAsObserverString := ctx.GlobalString(destinationShardAsObserver.Name)
+	if destinationShardAsObserverString != metachainShardName {
```

Comment:
You can compare this in insensitive mode using this line:

if !strings.EqualFold(destinationShardAsObserverString, metachainShardName)
or
if strings.ToLower(destinationShardAsObserverString) != strings.ToLower(metachainShardName)

Reply:
fixed